<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fakturka</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset resource="res/resources.qrc">
         <normaloff>:/profile.png</normaloff>:/profile.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Dane sprzedawcy</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Imię i nazwisko</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>NIP</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="seller_nip"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Konto bankowe</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="seller_account"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Bank</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="seller_bank"/>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Ulica i numer domu</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="seller_street"/>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Kod pocztowy i miasto</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="seller_city"/>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="seller_name"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset resource="res/resources.qrc">
         <normaloff>:/seller.png</normaloff>:/seller.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Dane kupującego</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Imię i nazwisko</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="buyer_bame"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>NIP</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="buyer_nip"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Ulica i numer domu</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="buyer_street"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Kod pocztowy i miasto</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="buyer_city"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="icon">
        <iconset resource="res/resources.qrc">
         <normaloff>:/receipt.png</normaloff>:/receipt.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Faktura</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Data sprzedaży</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Numer faktury</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Data wystawienia</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="invoice_nr"/>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="invoice_issued">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateEdit" name="invoice_term">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Typ płatności</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateEdit" name="invoice_selldate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Termin płatności</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="invoice_paymenttype">
            <property name="text">
             <string>Przelew</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset resource="res/resources.qrc">
         <normaloff>:/box.png</normaloff>:/box.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Towar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="4">
         <widget class="QTableWidget" name="table">
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Nazwa</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ilość</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Jednostka</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Cena netto</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>VAT %</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Wartość netto</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Wartość brutto</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="table_add">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res/resources.qrc">
            <normaloff>:/add.png</normaloff>:/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="table_remove">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res/resources.qrc">
            <normaloff>:/delete.png</normaloff>:/delete.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="table_pkwiu">
          <property name="text">
           <string>PKWiU</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="table_discount">
          <property name="text">
           <string>Rabat</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="icon">
        <iconset resource="res/resources.qrc">
         <normaloff>:/sent-mail.png</normaloff>:/sent-mail.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Generowanie/wysyłanie</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Nazwa pliku</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="generator_filename"/>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="generator_sendemail">
          <property name="text">
           <string>Wyślij na e-mail</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="generator_emailadress">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Tytuł</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="generator_emailtitle">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Treść</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QTextEdit" name="generator_emailcontent">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="generator_generate">
          <property name="text">
           <string>Generate</string>
          </property>
          <property name="icon">
           <iconset resource="res/resources.qrc">
            <normaloff>:/pdf.png</normaloff>:/pdf.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="icon">
        <iconset resource="res/resources.qrc">
         <normaloff>:/settings-gears.png</normaloff>:/settings-gears.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Ustawienia aplikacji</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_7">
        <item row="0" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_6">
          <item row="1" column="0">
           <widget class="QRadioButton" name="settings_smtpssl">
            <property name="text">
             <string>SMTP ssl</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="settings_smtptls">
            <property name="text">
             <string>SMTP tls</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="settings_smtpnotsecure">
            <property name="text">
             <string>SMTP nie zabezpieczone</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>SMTP host</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="settings_smtphost"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>SMTP port</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="settings_smtpport"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>SMTP nadawca</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="settings_smtpsender"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>SMTP użytkownik</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>SMTP hasło</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="inputMask">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGenerowanie"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGenerowanie">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>Generowanie</string>
   </property>
   <property name="toolTip">
    <string>Zapisz dane</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>table_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addRowToTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_remove</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeRowFromTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_pkwiu</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>togglePKWIUColumn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_discount</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleDiscountsColumn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>823</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>818</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generator_sendemail</sender>
   <signal>toggled(bool)</signal>
   <receiver>generator_emailadress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generator_sendemail</sender>
   <signal>toggled(bool)</signal>
   <receiver>generator_emailtitle</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generator_sendemail</sender>
   <signal>toggled(bool)</signal>
   <receiver>generator_emailcontent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addRowToTable()</slot>
  <slot>removeRowFromTable()</slot>
  <slot>togglePKWIUColumn(bool)</slot>
  <slot>toggleDiscountsColumn(bool)</slot>
 </slots>
</ui>
